### ISSUE TITLE
Feature Request: Set min value of slider

### ISSUE THREAD

**Issue by WebTiger89** [hearsilent/DiscreteSlider#8]
Hi,
can you implement an attribute for setting slider min value so that the slider starts for example at 23 instead of 0?
Thanks in advance!

**Comment by hearsilent** [comment]
Hi @WebTiger89 ,
I think maybe you can use `ValueLabelFormatter` to customize begin value with offset?

For example:
```java
mSlider.setValueLabelFormatter(new DiscreteSlider.ValueLabelFormatter() {

    @Override
    public String getLabel(int input) {
        return Integer.toString(input + 23);
    }
});
```

**Comment by WebTiger89** [comment]
Thanks for the reply.
I already thought about this solution but then I have to convert the numbers internally (0 is 23, 23 is 46) which is not a problem at all. It would be cool to have this feature out of the box but consider that the SeekBar widget from android doesn't have such a feature so its up to you if you will implement this. It would be just a nice to have :)

**Comment by hearsilent** [comment]
@WebTiger89 
I can implement this, but I'm busy this month.
Maybe you can create PR, I'll review & merge it 👍

**Comment by hearsilent** [comment]
@WebTiger89 I've released new version 1.2.0, you can try now!
